crowd-ui/src/App.css

```diff
@@ -1,39 +1,5 @@
-.App {
-  text-align: center;
-}
+ .App {
```

Comment:
why empty class?

Reply:
because this is a global CSS file and i don't know could i delete this or no?